scene/2d/camera_2d.cpp

```diff
@@ -39,8 +39,11 @@ void Camera2D::_update_scroll() {
 	}
 
 	if (Engine::get_singleton()->is_editor_hint()) {
-		queue_redraw(); //will just be drawn
-		return;
+		queue_redraw();
+		// Only set viewport transform when not bound to the main viewport.
+		if (get_viewport() == get_tree()->get_edited_scene_root()->get_viewport()) {
+			return;
+        }
```

Comment:
Indentation issue.

Reply:
Yeah, sorry. I thought I'd fixed everything and went to lunch. I've pushed a fixed version.